server/server.js

```diff
@@ -157,6 +158,10 @@ function server(serviceManager, options) {
       case 'express:usage-record':
         ExpressUsageRecord.recordUsage(instanceId, uInfo, callback);
         break;
+      case 'status:wd':
```

Comment:
I don't think the current working directory is known, and I don't think the PWD value is known if we don't do an explicit env-get command.